specification/common-types/resource-management/v6/networksecurityperimeter.json

```diff
@@ -0,0 +1,480 @@
+          },
+          {
+            "name": "Accepted",
+            "value": "Accepted"
+          },
+          {
+            "name": "Failed",
+            "value": "Failed"
+          },
+          {
+            "name": "Canceled",
+            "value": "Canceled"
+          }
+        ]
+      }
+    },
+    "NetworkSecurityProfile": {
+      "type": "object",
+      "description": "Network security perimeter configuration profile",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the profile"
+        },
+        "accessRulesVersion": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Current access rules version"
+        },
+        "accessRules": {
+          "type": "array",
+          "description": "List of Access Rules",
+          "items": {
+            "$ref": "#/definitions/AccessRule"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "diagnosticSettingsVersion": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Current diagnostic settings version"
+        },
+        "enabledLogCategories": {
+          "type": "array",
+          "description": "List of log categories that are enabled",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ProvisioningIssue": {
+      "type": "object",
+      "description": "Describes a provisioning issue for a network security perimeter configuration",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the issue",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/ProvisioningIssueProperties",
+          "readOnly": true
+        }
+      }
+    },
+    "ProvisioningIssueProperties": {
+      "type": "object",
+      "description": "Details of a provisioning issue for a network security perimeter (NSP) configuration. Resource providers should generate separate provisioning issue elements for each separate issue detected, and include a meaningful and distinctive description, as well as any appropriate suggestedResourceIds and suggestedAccessRules",
+      "properties": {
+        "issueType": {
+          "$ref": "#/definitions/IssueType",
+          "description": "Type of issue",
+          "readOnly": true
+        },
+        "severity": {
+          "$ref": "#/definitions/Severity",
+          "description": "Severity of the issue.",
+          "readOnly": true
+        },
+        "description": {
+          "type": "string",
+          "description": "Description of the issue",
+          "readOnly": true
+        },
+        "suggestedResourceIds": {
+          "type": "array",
+          "description": "Fully qualified resource IDs of suggested resources that can be associated to the network security perimeter (NSP) to remediate the issue.",
```

Comment:
>IDs [](http://example.com/codeflow?start=51&length=3)

style cleanup? looks like we prefer to call them 'identifiers' instead of 'IDs' for doc style?

Reply:
same as: https://github.com/Azure/azure-rest-api-specs/blob/2d3a19a9754e340760eed074ebd75987585ce0bd/specification/common-types/resource-management/v5/networksecurityperimeter.json#L353

let's have a discussion about all this, I would prefer to focus this particular pull request in just the migration, then we can have